src/cfml/system/util/CompileDSL.cfc

```diff
@@ -0,0 +1,402 @@
+
+/**
+*********************************************************************************
+* Copyright Since 2014 CommandBox by Ortus Solutions, Corp
+* www.coldbox.org | www.ortussolutions.com
+********************************************************************************
+* @description
+*
+* I am a helper object for executing compile actions for "javac" command and
+* jarring actions for "jar" command. Create me and call my
+* methods to compile and jar any java project, then .run() will execute me.
+*
+*/
+
+component accessors=true {
+
+    /* root folder of the project you are working with (compiling, jar) */
+    property name='projectRoot'             type="string";
+    /* folder inside root project where source files reside (.java) */
+    property name='sourceDirectory'         type='string';
+    property name='classPathDirectory'      type='string';
+    property name='classOutputDirectory'    type='string';
+    property name='verbose'                 type='boolean';
+    property name='encode'                  type='string';
+	property name='sourcePaths'				type='array';
+	property name='recursive'				type='boolean';
```

Comment:
Don't forget to remove this property and the corresponding setters and usage of it.

Reply:
done